cli/cli_app.go

```diff
@@ -309,7 +309,7 @@ func RunTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 	}
 
 	if terragruntConfig.Skip {
-		terragruntOptions.Logger.Infof("Skipping terragrunt module %s due to skip = true.",
+		terragruntOptions.Logger.Warnf("Skipping terragrunt module %s due to skip = true.",
```

Comment:
Why change this one to a warning?

Reply:
@brikis98 because [TestApplySkipTrue() expects](https://github.com/gruntwork-io/terragrunt/blob/master/test/integration_test.go#L2086) this output with default log level (which is "warn" in this PR).

Reply:
Ah, fair enough

Reply:
Actually, we should make that more explicit. You can keep this log statement an `Infof`, but in the `TestApplySkipTrue`, on [this line](https://github.com/gruntwork-io/terragrunt/blob/master/test/integration_test.go#L2078), set `--terragrunt-log-level info`. Good integration test of logging behavior too!